src/Plugin.php

```diff
@@ -82,6 +82,13 @@ public static function init() {
       apply_filters('bjll/skip_classes', __CLASS__ . '::bjll_skip_classes');
     }
 
+    // Deletes transient on product update.
+    if (static::isPluginActive('woocommerce/woocommerce.php')) {
```

Comment:
You don't need to check if the plugin is active when registering a hook. In case the plugin isn't active. the hook will not be triggered. Also, please use dedicated methods :)